### PR TITLE
fix: fix the style of version

### PR DESCRIPTION
**Pull request by likyh**
### Summary
fix the style of version

closes https://github.com/apache/incubator-devlake/issues/3607

### Screenshots
image

### CODE REVIEW

**Review by Startrekzky — APPROVED** [review]
The design is acceptable.

File: config-ui/src/layouts/base/styled.ts [hunk 1]
```diff
@@ -73,6 +73,12 @@ export const Sider = styled.div`
     left: 0;
     text-align: center;
     color: rgba(124, 124, 124, 0.7);
+    padding: 0 20px;
+    div {
```
Comment (mintsweet) [inline config-ui/src/layouts/base/styled.ts:77]: Don't use div directly to add a class.

File: config-ui/src/layouts/base/base.tsx [hunk 2]
```diff
@@ -76,8 +76,8 @@ export const BaseLayout = ({ children }: Props) => {
           ))}
         </Menu>
         <div className="copyright">
-          <div>{version}</div>
           <span>Apache 2.0 License</span>
+          <div>{version}</div>
```
Comment (mintsweet) [inline config-ui/src/layouts/base/base.tsx:80]: You can use `span` and add a class.

**Review by mintsweet — APPROVED** [review]
LGTM.

**Pull request merged**